src/components/QrCodeForm.vue

```diff
@@ -0,0 +1,208 @@
+<script setup lang="ts">
+import { pix } from "../utils/QrCodePix";
+import { ref } from "vue";
+import { debounce } from "@/utils/debounce";
+import CustomButton from "./CustomButton.vue";
+import api from "../services/index";
+
+const props = defineProps({
+  pixTarget: String,
+  tokenValue: Number,
+});
+
+console.log(props.tokenValue);
+console.log(props.pixTarget);
+
+const qrCode = ref<string>("");
+const qrCodePayload = ref<string>("");
+const pixQrCode = pix({
+  pixKey: props.pixTarget ?? "",
+  value: props.tokenValue,
+});
+const pixIsValid = ref<number>(0);
+
+pixQrCode.base64QrCode().then((code: string) => {
+  qrCode.value = code;
+});
+
+qrCodePayload.value = pixQrCode.payload();
+
+const handleInputEvent = (event: any) => {
+  const { value } = event.target;
+
+  validatePix(value);
+};
+
+const validatePix = async (value: any) => {
```

Comment:
You can replace this parameter name to e2eid: string. Because we know what's being passed and it will be more concise

Reply:
Also, remove all the console.log in this function pls